src/spurt/workflows/emcf/_cli.py

```diff
@@ -35,11 +35,18 @@ def main(args=None):
         "--tempdir", default="./emcf_tmp", help="Folder for intermediate outputs."
     )
     parser.add_argument(
-        "-w",
-        "--workers",
+        "-tw",
```

Comment:
If you use a single hyphen, the argument name should be a single alphanumeric character per [POSIX convention](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html)

This allows combining multiple options (`-abc` is equivalent to `-a -b -c`) or combining an option with its argument (`-o foo` and `-ofoo` are equivalent)

Reply:
Suggestions of better version? Or just scrap the single letter ones?

Reply:
I think your best options are:

1. Scrap the single-hyphen aliases
2. Change `-tw` back to `-w` and remove the single-hyphen alias from `--s-workers`
3. Change `-sw` to `-w` and remove the single-hyphen alias from `--t-workers`
4. Use `-t` for temporal workers and `-s` for spatial workers

I think I would probably go with option 2 (or else option 3 if we expect parallelizing the spatial unwrapping to be more profitable than parallelizing temporal unwrapping), but I'd be fine with any of the above